app/code/Magento/Directory/Model/ResourceModel/Country/Collection.php

```diff
@@ -68,6 +68,10 @@ class Collection extends \Magento\Framework\Model\ResourceModel\Db\Collection\Ab
      * @since 100.1.0
      */
     protected $countriesWithNotRequiredStates;
+    /**
+     * @var \Magento\Store\Model\StoreManagerInterface
+     */
+    protected $storeManager;
```

Comment:
Please avoid adding new `protected` methods and properties. This may be made private.